server/config/config.exs

```diff
@@ -60,13 +63,27 @@ config :realtime,
   configuration_file: configuration_file,
   secure_channels: secure_channels,
   jwt_secret: jwt_secret,
-  jwt_claim_validators: jwt_claim_validators
+  jwt_claim_validators: jwt_claim_validators,
+  ecto_repos: [Realtime.Repo]
 
 # Configures the endpoint
 config :realtime, RealtimeWeb.Endpoint,
   url: [host: "localhost"],
   render_errors: [view: RealtimeWeb.ErrorView, accepts: ~w(html json)],
-  pubsub: [name: Realtime.PubSub, adapter: Phoenix.PubSub.PG2]
+  pubsub_server: Realtime.PubSub,
+  secret_key_base: session_secret_key_base
+
+config :realtime, Realtime.Repo,
+  hostname: db_host,
+  port: db_port,
+  database: db_name,
+  username: db_user,
+  password: db_password
```

Comment:
These settings are currently the same as the realtime listerner here: 
https://github.com/supabase/realtime/blob/af6344c7746e8a8af6a11a9b498721c1f97e339b/server/lib/realtime/application.ex#L24

Even though they will be the same in many cases, it might be useful to split them up into a "workflows" database connection and a "realtime" database connection

The reasoning is that:

- Perhaps Postgres doesn't scale as a store for workflows + logs, so we have to switch to something else
- Perhaps in the future we want to split the "realtime" source into several inputs

![image](https://user-images.githubusercontent.com/10214025/107959340-40789800-6fde-11eb-9486-ca8278b9216d.png)


From our conversation this morning, it seems that you're listening to the changes in Postgres, using the same listener that can trigger the workflows - this separate config could prevent the user from triggering "infiinte" workflows (and we should defintiely namespace everything to a `workflows` schema inside the database)